src/Filesystem/Filesystem.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace SprykerSdk\Evaluator\Filesystem;
+
+use Symfony\Component\Filesystem\Exception\IOException;
+use Symfony\Component\Filesystem\Filesystem as SymfonyFilesystem;
+
+class Filesystem extends SymfonyFilesystem
+{
+    /**
+     * @var string|null
+     */
+    protected static ?string $lastError;
+
+    /**
+     * @param string $filename
+     * @param bool $useIncludePath
+     * @param mixed $context
+     *
+     * @throws \Symfony\Component\Filesystem\Exception\IOException
+     *
+     * @return string
+     */
+    public function readFile(
+        string $filename,
+        bool $useIncludePath = false,
+        $context = null
+    ): string {
+        $data = $context === null
+            ? static::box('file_get_contents', $filename, $useIncludePath)
+            : static::box('file_get_contents', $filename, $useIncludePath, $context);
+
+        if ($data === false) {
+            throw new IOException(sprintf('Failed to read "%s" file: %s', $filename, static::$lastError));
+        }
+
+        return $data;
+    }
+
+    /**
+     * @param string $func
+     * @param mixed ...$args
+     *
+     * @throws \Symfony\Component\Filesystem\Exception\IOException
+     *
+     * @return mixed
+     */
+    protected static function box(string $func, ...$args)
```

Comment:
It looks a bit like overengineering. This method is used only once for file_get_contents only.